db/migrate/20220323035529_add_column_to_user.rb

```diff
@@ -0,0 +1,6 @@
+class AddRoleToUser < ActiveRecord::Migration[7.0]
```

Comment:
- [ ] Kindly note that you don't have a table called `role` to run this migration that says ` AddRoleToUser` rather you should run a migration that adds a column to the user table. Hint :point_down: 

```rb
class AddColumnToUser < ActiveRecord::Migration[7.0]
 def change
    add_column :users, :role, :string,
    default: 'user'
  end
end
```
this will fixe this error I am getting 

![image](https://user-images.githubusercontent.com/65068771/159833958-fbbb3f68-f5da-4e61-b5ff-1ad412a1a583.png)